backend/blueprint/api/admin/users.py

```diff
@@ -0,0 +1,137 @@
+from flask import Blueprint, jsonify, request
+from api.utils.response import Response
+from firebase_admin import auth
+
+
+router = Blueprint('users', __name__)
+
+
+@router.route('/createuser', methods=['POST'])
+def create_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user = auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            email_verified = data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled = data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully created new user named {0} with user-id: {1}'.format(user.display_name, user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        print(e)
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/updateuser', methods=['POST'])
+def update_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user = auth.update_user(
+            data['uid'],
+            display_name=data['name'],
+            email=data['email'],
+            email_verified = data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled = data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully updated user: {0}'.format(user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/deleteuser', methods=['POST'])
+def delete_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        auth.delete_user(
+            data['uid']
+        )
+        return jsonify({
+            'message': 'Successfully deleted user',
```

Comment:
Please see the comments and change elsewhere.

Reply:
👍